f8a_worker/workers/stackaggregator_v2.py

```diff
@@ -86,21 +91,36 @@ def extract_component_details(component):
         "osio_user_count": component.get("osio_user_count", -1),
         "latest_version": latest_version,
         "github": github_details,
-        "code_metrics": code_metrics
+        "code_metrics": code_metrics,
     }
-    return component_summary, licenses
+    return component_summary, licenses, license_dict
```

Comment:
That's a very good catch. I was just modifying on top of other code, but let me rework on this.